### DISCUSSION TITLE
@jagrosh How to create the playlists on the bot?

### DISCUSSION THREAD

**Discussion by ek123b** [jagrosh/MusicBot#1319] [Questions]
How to create the playlists on the bot?
@jagrosh

**Comment by jagrosh** [comment]
Information about playlists can be found on the Playlists page: <https://jmusicbot.com/playlists/>. If this page doesn't cover what you need, consider reading the Advanced Playlist Generation page: <https://jmusicbot.com/advanced-playlists/>

**Reply by ek123b** [reply]
k
